ale/drivers/cassini_drivers.py

```diff
@@ -90,3 +86,13 @@ def odtk(self):
         elif self.instrument_id == 'CASSINI_ISS_NAC':
             # NAC
             return [float('-8e-6'), 0, 0]
+
+    @property
+    # Don't know which (n, n) value from the FOV_CENTER_PIXEL is samples or lines
```

Comment:
Any way to confirm which is which?

Reply:
For Cassini, in the current IK, the x and y coordinates are the same so swapping them wouldn't change anything.

That being said, there isn't anything in the IK that specifies the order, so @scsides suggested that I make a note of this. I'm not sure where else to look besides the IK for this.

Reply:
Ok, documenting and moving on seems like the way to go.